Sources/Bot/Services/DispatchService.swift

```diff
@@ -43,9 +43,6 @@ public final class DispatchService {
 
         gitHubAPI.fetchPullRequests()
             .flatMapError { _ in .value([]) }
-            .map { pullRequests in
-                pullRequests.filter { $0.isLabelled(with: self.integrationLabel) }
-            }
```

Comment:
`DispatchService` shouldn't know about the logic related to deciding which PR to filter in or out. That filtering will be done by the `MergeService` instead to avoid leaking any impl details and logic of the `MergeService` to the `DispatchService` (even if that means creating a `MergeService` only to see it immediately go to its `.idle` state and being destroyed.